starfish/spots/_detector/blob.py

```diff
@@ -133,6 +133,7 @@ def run(
             primary_image: ImageStack,
             blobs_image: Optional[ImageStack] = None,
             blobs_axes: Optional[Tuple[Axes, ...]] = None,
+            n_processes: Optional[int] = None,
```

Comment:
I don't think you need the = None if it's an optional param

Reply:
You do still need it.

Strictly speaking, `Optional` is optional, because mypy developers have decided to relax the rules for now.

Reply:
Can you say more / link to the documentation that defines the valid uses (or lack of uses) of `Optional`?

Reply:
https://mypy.readthedocs.io/en/latest/kinds_of_types.html#no-strict-optional

Apparently the optional Optional is gone though, except in the case of a parameter that defaults to None.